### DISCUSSION TITLE
Components and Serial Numbers

### DISCUSSION THREAD

**Discussion by JakeHagemann** [snipe/snipe-it#10019] [Q&A]
It seems like you can't keep track of components by serial numbers.  If I wanted to track most of my Items by serial number would they all have to be Assets? 

Thank you

**Comment by uberbrady** [comment]
*Probably*, yes. Generally if you care about "which _one_ of these things, is this thing?" Then it's an asset, not a component.

If you have a drawer full of RAM, let's say, and you just care about what type of RAM (who makes it) and how much there is, and so on - that's closer to what we probably mean by Components.

At least, that's how I'd use it. We get a lot of folks using our stuff a lot of weird ways :)

**Reply by JakeHagemann** [reply]
Thank you for the quick reply and for answering my question. Git or my browser is being a pain and it will not let mark this as answered. 
Unfortunate to hear on my end because everything is an asset. We swap around a bunch of hardware from server to server for performance testing. Maintaining a history for each component would be a really nice feature to have.

**Reply by snipe** [reply]
> Unfortunate to hear on my end because everything is an asset.

If everything is an asset, you'd automatically be maintaining history, no? You can check out assets to other assets, so in your case, I'd just make the components into assets, check them out to other assets, and then you have full history.
